src/transaction/TransactionController.ts

```diff
@@ -102,12 +104,27 @@ export enum TransactionStatus {
   cancelled = 'cancelled',
   confirmed = 'confirmed',
   failed = 'failed',
+  failedBeforeChain = 'failedBeforeChain',
```

Comment:
Remove this if not used in the current solution.

Reply:
Done